app/assets/stylesheets/reskin/moneyhelper_styling_common.scss

```diff
@@ -0,0 +1,210 @@
+// -- Headers ----------------------------------------------------------------------
+h1,h2,h3,h4,h5,h6{
+    font-family: 'Roobert-Bold', 'Times';
+    font-style: normal;
+    font-weight: bold;
+    color: $blue_primary;
+}
+
+// ---------------------------------------------------------------------------------
+
+
+// -- Paragraphs -------------------------------------------------------------------
+p{
+    font-family: 'Roobert-Regular', sans-serif!important;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 23px;
+    color: $text_secondary;
+    &.callout{
+        background-color: $light_grey;
+        border-radius: $border_radius;
+        padding: 10px;
+    }
+}
+// ---------------------------------------------------------------------------------
+
+// -- Links ------------------------------------------------------------------------
+    a:not(.navigation__link){
+        color: $magenta_primary!important;
```

Comment:
Can we try to avoid the `!important` value here and elsewhere? 
In this instance we would probably be better to define a universal colour for all `</a>` tags and specify a separate rule for `.navigation__link`.